.github/workflows/dist.yml

```diff
@@ -61,3 +61,52 @@ jobs:
           name: input-leap-rpms-${{ matrix.rpm_tag }}
           path: |
             rpms/**/*.rpm
+
+  flatpak:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./input-leap
+    steps:
+      # submodules require git to be installed
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          path: input-leap
+          # submodules: recursive
+          set-safe-directory: ${{ github.workspace }}
+
+      - run: git config --global protocol.file.allow always
+
+      - name: Install flatpak and flatpak builder
+        run: sudo apt-get install flatpak flatpak-builder
+
+      - name: add flatpak remote
+        run: sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+
+      - name: Install flatpak dependencies
+        run: sudo flatpak install -y flathub org.kde.Sdk//5.15-23.08 org.kde.Platform//5.15-23.08
+
+      - name: make the build dir
+        run: mkdir build-dir
+
+      - name: build the flatpak
+        run: flatpak-builder --force-clean --repo repo-dir build-dir dist/flatpak/com.github.input-leap.input-leaps.yml
+
+      # - name: test
+      #   run: flatpak-builder --run build-dir dist/flatpak/com.github.input-leap.input-leaps.yml
+
```

Comment:
this should be removed, this can't ever work on the CI anyway since we have neither X nor a compositor running (technically we could emulate one with dbusmock but meh)

Reply:
removed/fixed